### DISCUSSION TITLE
In cmake project, intellisense only works after build

### DISCUSSION THREAD

**Discussion by playgithub** [microsoft/vscode-cpptools#7957] [Q&A]
## Env
OS: Linux
Compiler: g++ / clang
vscode: 1.59.0

## Reproduce Problem
1. Open a folder with a cmake project, C/C++ intellisense not work, e.g.
    * goto include file
    * auto completion
2. Build the project, then intellisense works
3. Close vscode and reopen the folder, intellisense doens't work again

## Question
* How to let intellisense for C/C++ work automatically?

**Comment by sean-mcmanus** [comment]
What version of the C/C++ extension are you using?

Are you using the CMake Tools extension with the configurationProvider set to "ms-vscode.cmake-tools"? Are you running CMake: Configure? Does your project generate required headers after building?

Can you provide the logs from running the C/C++: Log Diagnostics command after opening a file?

**Reply by playgithub** [reply]
In fact, I'm using vscodium and extenstions from its own extenstion market, the extensions are up to date.

**Reply by sean-mcmanus** [reply]
vscodium may not have access to the most recent versions of VS Code or our extension.

**Reply by playgithub** [reply]
ok
